.github/workflows/python-app.yml

```diff
@@ -36,6 +36,26 @@ jobs:
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
         
+          
+    - name: Test using unittest
+      run: |
+        python -m unittest
+      env:
+        TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+        DATABASE_URL: "${{ secrets.DATABASE_URI}}"
+        EMAIL: ${{ secrets.EMAIL }}
+        SECRET_KEY: ${{ secrets.SECRET_KEY }}
```

Comment:
There is already a Test using unittest block on the staging branch. Did you checkout this branch from a recent staging commit? I think you need to rebase this branch

Reply:
Yes, I did. The unittest step was after the slack notification step. Just repositioned it only.